action.go

```diff
@@ -343,7 +343,7 @@ func (a Action) String() (str string, err error) {
 // permission is found, the default one `default` is used.
 // The first permission that is found to apply to an operation, but
 // doesn't allow the operation to run, will fail the verification globally
-func (a Action) VerifyACL(acl ACL, keyring io.Reader) (err error) {
+func (a Action) VerifyACL(acl ACL, keyring io.Reader, verifyACLs bool) (err error) {
```

Comment:
I just committed [9ad057e](https://github.com/jboyer2012/mig/commit/9ad057e22962def33050b75aea3f40b7aeaf7519) for these changes. I decided to change the flag everywhere. After thinking about it, I feel it is clearer in its current form. I also reworded the comments to make more explicit what the flag does. Let me know if you like the changes.